specs-go/config.go

```diff
@@ -53,15 +53,16 @@ type Process struct {
 	SelinuxLabel string `json:"selinuxLabel,omitempty" platform:"linux"`
 }
 
-// User specifies Linux/Solaris specific user and group information
-// for the container process.
+// User specifies specific user (and group) information for the container process.
 type User struct {
 	// UID is the user id. (this field is platform dependent)
 	UID uint32 `json:"uid" platform:"linux,solaris"`
 	// GID is the group id. (this field is platform dependent)
 	GID uint32 `json:"gid" platform:"linux,solaris"`
 	// AdditionalGids are additional group ids set for the container's process. (this field is platform dependent)
 	AdditionalGids []uint32 `json:"additionalGids,omitempty" platform:"linux,solaris"`
+	// User is the user name. (this field is platform dependent)
+	User string `json:"user,omitempty" platform:"windows"`
```

Comment:
To avoid `User.User`, can we call this `Username` or something?  And in that case it could be either `Process.User.Username` or `Process.Username` instead of `Process.User.User`.  I like `Process.Username` best, as long as Windows doesn't plan on needing a more complicated structure in the future (in which case it's probably better to stay under `Process.User`).